.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is nice to see you are configuring your editor but be carefull when commiting. This is an editor config file and should not be commited under any circumstance. Can you try to add it to the .gitignore file?